nautilus_open_any_terminal/nautilus_open_any_terminal.py

```diff
@@ -226,27 +227,28 @@ def menu_item_id(*, foreground: bool, remote: bool):
     return f"OpenTerminal::open{'_' if foreground else '_bg_'}{'remote' if remote else 'file'}_item"
 
 
-def get_menu_items(file: FileManager.FileInfo, callback, foreground: bool):
+def get_menu_items(file: FileManager.FileInfo, callback, *, foreground: bool, terminal_name: str | None = None):
     items = []
     remote = file.get_uri_scheme() in REMOTE_URI_SCHEME
+    terminal_name = terminal_name or terminal_data.name
```

Comment:
can we use a different variable name for this?
I know that python had some linter issues with reassigning to the same naming but different types.

```suggestion
    terminal_str = terminal_name or terminal_data.name
```

do not forget the var below

Reply:
would declaring the type explicitly also work?

Reply:
yeah might be sufficient as well.

Reply:
this shouldn't be a problem since string can be assigned to `str | None` just fine

Reply:
making a new variable here looks really ugly, i'd rather just keep this as it is and if a linter can't handle assigning a default value to a function parameter then it's broken, i shouldn't have to work around someone else's broken software